src/experiments.ts

```diff
@@ -25,8 +26,9 @@ export class ExperimentalSettings {
         private readonly context: vscode.ExtensionContext,
         private readonly logger: ILogService
     ) {
+        const isProduction = this.context.extension.packageJSON.buildEnv === 'production';
```

Comment:
Why do we need another flag in package json? vs code context already have it?

Reply:
https://github.com/gitpod-io/gitpod-vscode-desktop/pull/36#discussion_r1168560639

To avoid install extension locally send debugging telemetry to production

Reply:
You can put an env var like `VSCODE_DEV: 1` instead? to avoid messing with package.json during release?

Reply:
> You can put an env var like VSCODE_DEV: 1 instead? 

Do you mean configure `VSCODE_DEV` in my `.zshrc`? I don't want to mess my local environment.

> to avoid messing with package.json during release?

1️⃣ I added `setBuildEnv="setpath([\"buildEnv\"]; \"production\")"` to `.github/workflow/nightly.yaml` and `.github/workflow/release.yaml` is it enough?

2️⃣ Or I'd like to stash these changes, and apply when I'm going to debug with locally installation.

Which one do you prefer?